tiapp.xml

```diff
@@ -4,14 +4,14 @@
         <target device="ipad">true</target>
         <target device="android">true</target>
     </deployment-targets>
-    <sdk-version>12.0.0.GA</sdk-version>
+    <sdk-version>12.6.1.GA</sdk-version>
     <id>org.piwik.mobile2</id>
     <name>Matomo Mobile 2</name>
-    <version>2.5.2</version>
+    <version>2.5.5</version>
     <publisher>thomassteur</publisher>
     <url>https://matomo.org/mobile</url>
     <description>Need your analytics on the go? Use our free Matomo Mobile (formerly Piwik Mobile) app to check your stats from anywhere and show them to colleagues in meetings. Matomo Mobile offers all the same reports as the Matomo (Piwik) desktop interface, and displays your visitor data in real time!</description>
-    <copyright>2023 by matomo</copyright>
+    <copyright>2010 by matomo</copyright>
```

Comment:
This probably should stay at 2023 or even 2024?

Reply:
Or is it intentionally going back to when Matomo started?

Reply:
it's intentional